package exception;

public  class TestFinal {
public final void test() {}
}
